index.html

```diff
@@ -25,5 +25,99 @@ <h3 class="text2">I'm a Software Developer</h3>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="second">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav>
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+        </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="third">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="fourth">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="fifth">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        </div>
+    </section>
+  <section class="about-me">
+    <h2 class="about">About me</h2>
+    <p class="aboutme">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <nav class="button1"><img src="icons/Enable.png">
```

Comment:
This is suppose to be a button or a link `a` tab and not an image. Kindly use the button or link tag to create it. 
You also need to implement all the button styling interactions (for all your buttons) like the Active, Hover and Pressed as shown in the design below :point_down: 

![Screenshot from 2023-04-29 07-39-21](https://user-images.githubusercontent.com/65233860/235291245-69d5f17f-1e06-483b-8744-9e69425b4258.png)